apps/dashboard/src/pages/api-keys.tsx

```diff
@@ -0,0 +1,149 @@
+import { useState } from 'react';
+import { RiKey2Line, RiEyeLine, RiEyeOffLine, RiLockStarFill, RiExternalLinkLine } from 'react-icons/ri';
+import { useEnvironment } from '@/context/environment/hooks';
+import { CopyButton } from '@/components/primitives/copy-button';
+import { Card, CardContent } from '@/components/primitives/card';
+import { Button } from '@/components/primitives/button';
+import { InputField } from '@/components/primitives/input';
+import { Form } from '@/components/primitives/form/form';
+import { useForm } from 'react-hook-form';
+import { DashboardLayout } from '../components/dashboard-layout';
+import { PageMeta } from '@/components/page-meta';
+import { useApiKeysQuery } from '../hooks/use-api-keys-query';
+
+interface ApiKeysFormData {
+  apiKey: string;
+  environmentId: string;
+  identifier: string;
+}
+
+export function ApiKeysPage() {
+  const apiKeysQuery = useApiKeysQuery();
+  const { currentEnvironment } = useEnvironment();
+  const [showApiKey, setShowApiKey] = useState(false);
+  const apiKeys = apiKeysQuery.data?.data;
+
+  const form = useForm<ApiKeysFormData>({
+    values: {
+      apiKey: apiKeys?.[0]?.key ?? '',
+      environmentId: currentEnvironment?._id ?? '',
+      identifier: currentEnvironment?.identifier ?? '',
+    },
+  });
+
+  if (!currentEnvironment) {
+    return null;
+  }
+
+  const toggleApiKeyVisibility = () => {
+    setShowApiKey(!showApiKey);
+  };
+
+  const maskApiKey = (key: string) => {
+    return `${'•'.repeat(28)}${key.slice(-4)}`;
+  };
+
+  return (
+    <>
+      <PageMeta title="Environment Keys" />
+      <DashboardLayout headerStartItems={<h1 className="text-foreground-950">API Keys</h1>}>
+        <div className="flex flex-col gap-6 p-2">
+          <div className="grid grid-cols-1 gap-6 pt-10 lg:grid-cols-[1fr,500px]">
```

Comment:
```suggestion
          <div className="grid grid-cols-1 gap-6 lg:grid-cols-[1fr,500px]">
```

The padding-top pushes the section down  but creates a bit gap.


<img width="1673" alt="Screenshot 2024-12-04 at 15 50 48" src="https://github.com/user-attachments/assets/a576d8a2-cde3-4ef6-9d70-52de4cb69e49">